docs/installation-and-operation/observability-with-prometheus.md

```diff
@@ -0,0 +1,158 @@
+---
+title: observability-with-prometheus
+---
+
+# Observability with Prometheus
+
+You can export [Prometheus](https://prometheus.io/) stats from your Metabase.
+
+## Running Metabase and Prometheus locally
+
+To give you an idea of how Metabase and Prometheus would work in your production environment, we'll walk through how to set up Metabase and Prometheus locally. 
+
+### Start up Metabase with `MB_PROMETHEUS_SERVER_PORT`
+
+Download the latest [Metabase JAR](https://www.metabase.com/start/oss/), and run Metabase using an environment variable to specify the Prometheus server port:
+
+```
+MB_PROMETHEUS_SERVER_PORT=9191 java -jar metabase.jar
+```
+
+The `MB_PROMETHEUS_SERVER_PORT=9191` specifies which port (`9191`) Metabase will use to send data to Prometheus. To clarify the ports that will be involved here:
+
+- Port `3000` is the port Metabase uses to serve the Metabase app. You can set another port with `MB_JETTY_PORT` (e.g., `MB_JETTY_PORT=3001`).
+- Port `9191` (or whichever port you specified with the `MB_PROMETHEUS_SERVER_PORT` environment variable) is the port Metabase uses to send metrics to Prometheus.
```

Comment:
I would say here that 9191 is the port where Prometheus will scrape metrics from Metabase (apps don't push metrics to Prometheus, it's the other way around)